locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/networking/JcloudsLocationSecurityGroupCustomizer.java

```diff
@@ -201,6 +229,47 @@ public JcloudsLocationSecurityGroupCustomizer addPermissionsToLocation(final Jcl
             return this;
         }
     }
+    /**
+     * Removes the given security group permissions from the given node with the given compute service.
+     * <p>
+     * Takes no action if the compute service does not have a security group extension.
+     * @param permissions The set of permissions to be removed from the location
+     * @param location Location to remove permissions from
+     */
+    public void removePermissionsFromLocation(final JcloudsMachineLocation location, final Iterable<IpPermission> permissions) {
```

Comment:
@Graeme-Miller should include `@since 0.9.0` when adding public methods like this.